firebase-firestore/src/main/java/com/google/firebase/firestore/local/MemoryMutationQueue.java

```diff
@@ -154,12 +156,16 @@ public MutationBatch addMutationBatch(
           .addToCollectionParentIndex(mutation.getKey().getPath().popLast());
     }
 
+    statsCollector.recordRowsWritten(MemoryMutationQueue.TAG, 1);
+
     return batch;
   }
 
   @Nullable
   @Override
   public MutationBatch lookupMutationBatch(int batchId) {
+    statsCollector.recordRowsRead(MemoryMutationQueue.TAG, 1);
```

Comment:
Can't this just be `TAG`? Also even if you don't want to name it just that, it's actually `MutationQueue.TAG`.